### PR TITLE
Add scripts for generating ASCII animations

### PR DESCRIPTION
**Pull request by prescod**
Sill very rough but worth a review or adoption.

https://asciinema.org/a/Z4HxV5IXOanYi8r2oWCCB2VhT

### CODE REVIEW

File: utility/animation/animation.sh [hunk 1]
```diff
@@ -0,0 +1,250 @@
+
+set -e
+
+typingspeed=10
+shortpause=1
+mediumpause=2
+longpause=5
```
Comment (jstvz) [inline utility/animation/animation.sh:7]: 🤔: If we pass `--idle-time-limit 2.5` to asciinema, then is `$longpause` limited to 2.5s?

File: utility/record-animation.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+cp $REPO/utility/animation/* $TMPDIR
+git clone https://github.com/prescod/CCI-Food-Bank.git
+cd CCI-Food-Bank
+cci org default --unset dev
+cci org scratch_delete dev
+cci org scratch_delete qa
+cd ..
+echo "Recording script in '$TMPDIR'"
+sleep 5
+time asciinema rec demo.cast -i 2.5 -c "bash $TMPDIR/animation.sh"
```
Comment (jstvz) [inline utility/record-animation.sh:22]:
```suggestion
time asciinema rec demo.cast --idle-time-limit 2.5 --command "bash $TMPDIR/animation.sh"
```

**Comment by davisagli** [comment]
@prescod did these end up in a different repository?

**Comment by prescod** [comment]
@davisagli Yes, the latest code is in:

https://github.com/SFDO-Tooling/cci-demo-animations

**Pull request closed without merging**